app/components/ak-breadcrumbs/container/index.hbs

```diff
@@ -0,0 +1,15 @@
+{{! @glint-nocheck: or}}
+<AkStack
```

Comment:
exposing `@direction,  @justifyContent, @alignItems` can affect breadcrumbs design and why not use ak-list ?

Reply:
AkStack is a much better option. If I use ak-list, I would have to write custom styles to achieve the "row" alignment of items. I would just remove the unnecessary stack properties from the container.